src/modeler/AutoRest.Swagger/SwaggerModeler.cs

```diff
@@ -293,20 +293,13 @@ public static string GetMethodName(Operation operation)
                 throw new ArgumentNullException("operation");
             }
 
-            if (operation.OperationId == null)
-            {
-                return null;
-            }
-
-            if (operation.OperationId.IndexOf('_') == -1)
-            {
-                return operation.OperationId;
-            }
-
-            var parts = operation.OperationId.Split('_');
-            return parts[1];
+            return GetMethodNameFromOperationId(operation.OperationId);
         }
 
+        public static string GetMethodNameFromOperationId(string operationId) => 
+            (operationId?.IndexOf('_') != -1) ? operationId.Split('_')[1]: operationId;
```

Comment:
Do we always assume that all operationIds MUST have a single underscore? I know we have a validation rule to enforce that but still seems like a sensitive change.